tensorboard/defs/defs.bzl

```diff
@@ -200,3 +200,24 @@ def tf_ng_module(assets = [], **kwargs):
         angular_assets = assets,
         **kwargs
     )
+
+def tf_inline_images(name, html_template, images, out):
```

Comment:
I forgot that the utility literally only inlines PNG. We can generalize that and keep current name (by using mimetypes Python module to ascertain the MIME information) and replace these lines (`MIME_TYPE` part) or rename this rule to be `tf_inline_pngs`. WDYT?

https://github.com/tensorflow/tensorboard/blob/ba778d70baf2754932914d2715dfce6e5eb80dde/tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive/inline_images.py#L27-L38

Reply:
Good point! Renaming to `tf_inline_pngs` for now and will generalize it later (added a TODO).